pontoon/base/static/css/style.css

```
@@ -197,6 +197,11 @@ th {
   color: var(--white-1);
 }
 
+.table-sort th.no-sort:hover {
+  color: var(--light-grey-7);
+  cursor: default;
```

```suggestion
  pointer-events: none;
```

With that, you can revert the changes made to [table.js](https://github.com/mozilla/pontoon/pull/3558/files#diff-79b6a0398959ea93fb1e1c5c9028bdfbca94d1e7d2d884a9ab3e4024e89add95).

Seems like `color` is not needed?

This change should be moved to table.css though, because we might needed it elsewhere.

---

Interesting, never seen `pointer-events` before. I tried it but it doesn't seem to work completely: 
* The pointer doesn't change (so I can replace both rules).
* The JS event is still triggered, so I still need the code in `table.js`.

> This change should be moved to table.css though, because we might needed it elsewhere.

That's one confusing bit I discovered: CSS seems to be in both files? `.table-sort th:hover` is literally the rule above this one. Will leave it here for now, especially if you plan to make other CSS changes.

---

The `:hover` bit needs to be deleted from the CSS selector.

---

I also edited the wrong part of the CSS 🤦🏼